05-deploying_shiny_app.Rmd

```diff
@@ -1,279 +1,239 @@
-# Deploying a Shiny App
+# Deploying a shiny app
 
-The following sumamrizes the steps to deploy a Shiny app are as follows:
+## Basic deployment
 
-1. Copy the [template project](https://github.com/moj-analytical-services/rshiny-example) within Github to a new repository, with a name of your choice.
-2. In R Studio, checkout this copy (file -> new project -> version control -> git).   (TODO:  Remember to make sure the template has a Rproj that enables packrat)
-3. Work on your Shiny app, using the template `ui.R` and `server.R` files.
-4. Ensure that packrat is enabled and your package dependencies are snapshotted using `packrat::snapshot()`
-5. Set the desired access permissions in `deploy.json` (i.e. whether it should be available for DOM1, Quantum, or external).
-6. When you're ready to share it, in GitHub create a 'release' and it will deploy in a few minutes.
-7. Ask the Platform team to add and remove users from your app in the [#ap_admin_request](https://asdslack.slack.com/messages/CBLAGCQG6/) Slack channel.
+### Summary
 
-Step-by-step instructions are below.
+In order to deploy a Shiny app, you should follow these steps:
 
-## Step-by-step guide to deploying an app
+1.  Copy the app template
+2.  Create a new webapp
+3.  Clone the repository
+4.  Develop the app
+5.  Manage dependencies
+6.  Set access permissions
+7.  Create a release in GitHub
+8.  Deploy in Concourse
+9.  Add users to the app
+10. Access the app
 
-### Copy the template project into a new Github repository
+### Copy the app template
 
-1. Begin by making a **copy** of the R Shiny template project on Github: https://github.com/new/import.
+1.  Go to [github.com/new/import](https://github.com/new/import).
+2.  Enter `https://github.com/moj-analytical-services/rshiny-template` in the field titled 'Your old repository’s clone URL'.
+3.  Select `moj-analytical-services` as the owner of the new repository.
+4.  Add a name for the new repository.
+5.  Set the privacy of the new repository to private.
+6.  Select 'Begin import'.
 
-2. Enter `https://github.com/moj-analytical-services/rshiny-template` in the input box entitled 'your old repository’s clone URL:'
+### Create a new webapp
 
-3. Ensure the 'owner' of the new repository is 'moj-analytical-services' and choose a name for your repository:
+Standard users are not able to create new webapps or webapp data sources themselves.
```

Comment:
What's the purpose of needing an admin for this step? We should explain rules, so users' appreciate they are not stupid beaurocracy and can suggest improvements.

Also, I assume the user can start creating their app while they wait. They just won't be able to share the deployed app with customers until this step is done (step: "Add users to the app") - is that right?

Reply:
I think the reason for needing an admin in this step is so we don't end up with tons of apps that we don't know about.

Users have been developing their apps before completing this step but have been using standard warehouse data sources as opposed to app data sources. They've therefore had to copy everything across later on. It's fairly quick and straightforward, so I think it makes sense for users to do it as one of the first steps.

Reply:
Makes sense. I've started a discussion to add more to this in another PR - let's not hold up this PR with this, since it's all an improvement already.